source/endpoints/ruleset.c

```diff
@@ -0,0 +1,878 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/common/array_list.h>
+#include <aws/common/hash_table.h>
+#include <aws/common/json.h>
+#include <aws/common/ref_count.h>
+#include <aws/common/string.h>
+#include <aws/sdkutils/endpoints/private/ruleset_types_impl.h>
+
+#define SKIP_DOC true
```

Comment:
what is SKIP_DOC?

Reply:
removed. left overs from my testing that i forgot to clean up